### PR TITLE
Migrate to npm from yarn

### PR DESCRIPTION
**Pull request by pawalt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 Official React-based website for Penn Labs' organizational information, member/alumni listings, product information, documentation etc. 
 
 ## Installation
-1. Install dependencies using `yarn` in the project directory.
-2. Run application using `yarn next`.
+1. Install dependencies using `npm install` in the project directory.
+2. Run application using `npm dev`.
```
Comment (cacabo) [inline README.md]:
```suggestion
2. Run application using `npm run dev`.
```

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "dev": "next",
     "build": "next build",
-    "start": "next",
+    "start": "next start",
```
Comment (cacabo) [inline package.json:29]: just making sure—does this command work?

**Review by cacabo — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (pawalt) [reply package.json:29]: It does. Tested locally. Doesn't build first

**Review by ArmaanT — APPROVED** [review]
LGTM!

**Pull request merged**